rpc/documentation/public_endpoints/getnodestats.md

```diff
@@ -6,43 +6,47 @@ None
+| `inbound.getblocks`              | u64  | The number of all received GetBlocks messages                                |
+| `inbound.getmemorypool`          | u64  | The number of all received GetMemoryPool messages                            |
+| `inbound.getpeers`               | u64  | The number of all received GetPeers messages                                 |
+| `inbound.getsync`                | u64  | The number of all received GetSync messages                                  |
+| `inbound.memorypool`             | u64  | The number of all received MemoryPool messages                               |
+| `inbound.peers`                  | u64  | The number of all received Peers messages                                    |
+| `inbound.pings`                  | u64  | The number of all received Ping messages                                     |
+| `inbound.pongs`                  | u64  | The number of all received Pong messages                                     |
+| `inbound.syncs`                  | u64  | The number of all received Sync messages                                     |
+| `inbound.syncblocks`             | u64  | The number of all received SyncBlock messages                                |
+| `inbound.transactions`           | u64  | The number of all received Transaction messages                              |
+| `inbound.unknown`                | u64  | The number of all received Unknown messages                                  |
+| `internal_rtt.getpeer`           | f64  | The average internal RTT (query to response) for GetPeer messages in seconds |
+| `internal_rtt.getsync`           | f64  | The average internal RTT for GetSync messages in seconds                     |
+| `internal_rtt.getblocks`         | f64  | The average internal RTT for GetBlocks messages in seconds                   |
+| `internal_rtt.getmemorypool`     | f64  | The average internal RTT for GetMemoryPool messages in seconds               |
+| `misc.block_height`              | u32  | The current block height of the node                                         |
+| `misc.blocks_mined`              | u32  | The number of blocks the node has mined                                      |
+| `misc.duplicate_blocks`          | u64  | The number of duplicate blocks received                                      |
+| `misc.duplicate_sync_blocks`     | u64  | The number of duplicate sync blocks received                                 |
+| `outbound.all_successes`         | u64  | The number of successfully sent messages                                     |
+| `outbound.all_failures`          | u64  | The number of failures to send messages                                      |
+| `queues.inbound`                 | u32  | The number of messages queued in the common inbound channel                  |
+| `queues.outbound`                | u32  | The number of messages queued in the individual outbound channels            |
```

Comment:
this reminds me; I forgot to add the description of the new queues from https://github.com/AleoHQ/snarkOS/pull/1067; perhaps you could include them here as a drive-by :innocent:?

Reply:
Done 👍